src/frequenz/sdk/timeseries/_quantities.py

```diff
@@ -0,0 +1,310 @@
+
+        Args:
+            other: The other quantity.
+
+        Returns:
+            Whether this quantity is greater than another.
+        """
+        return self._base_value > other._base_value
+
+    def __ge__(self, other: Self) -> bool:
+        """Return whether this quantity is greater than or equal to another.
+
+        Args:
+            other: The other quantity.
+
+        Returns:
+            Whether this quantity is greater than or equal to another.
+        """
+        return self._base_value >= other._base_value
+
+    def __lt__(self, other: Self) -> bool:
+        """Return whether this quantity is less than another.
+
+        Args:
+            other: The other quantity.
+
+        Returns:
+            Whether this quantity is less than another.
+        """
+        return self._base_value < other._base_value
+
+    def __le__(self, other: Self) -> bool:
+        """Return whether this quantity is less than or equal to another.
+
+        Args:
+            other: The other quantity.
+
+        Returns:
+            Whether this quantity is less than or equal to another.
+        """
+        return self._base_value <= other._base_value
+
+    def __eq__(self, other: object) -> bool:
+        """Return whether this quantity is equal to another.
+
+        Args:
+            other: The other quantity.
+
+        Returns:
+            Whether this quantity is equal to another.
+        """
+        if not isinstance(other, self.__class__):
+            return NotImplemented
+        return self._base_value == other._base_value
+
+
+class Power(
+    Quantity,
+    exponent_unit_map={
+        -3: "mW",
+        0: "W",
+        3: "kW",
+        6: "MW",
+    },
+):
+    """A power quantity."""
+
+    @classmethod
+    def from_watts(cls, watts: float) -> Self:
+        """Initialize a new power quantity.
+
+        Args:
+            watts: The power in watts.
+
+        Returns:
+            A new power quantity.
+        """
+        return cls(value=watts, exponent=0)
+
+    @classmethod
+    def from_milliwatts(cls, milliwatts: float) -> Self:
+        """Initialize a new power quantity.
+
+        Args:
+            milliwatts: The power in milliwatts.
+
+        Returns:
+            A new power quantity.
+        """
+        return cls(value=milliwatts, exponent=-3)
+
+    @classmethod
+    def from_kilowatts(cls, kilowatts: float) -> Self:
+        """Initialize a new power quantity.
+
+        Args:
+            kilowatts: The power in kilowatts.
+
+        Returns:
+            A new power quantity.
+        """
+        return cls(value=kilowatts, exponent=3)
+
+    @classmethod
+    def from_megawatts(cls, megawatts: float) -> Self:
+        """Initialize a new power quantity.
+
+        Args:
+            megawatts: The power in megawatts.
+
+        Returns:
+            A new power quantity.
+        """
+        return cls(value=megawatts, exponent=6)
+
+    def as_watts(self) -> float:
```

Comment:
no,  `as_` is a function.  just `quantity.watts` is a property.  we can discuss later when I make the PR ready for review.

Reply:
OK, not sure what do you mean about `quantity.watts` as we said we didn't want to call it `watts` because it can be more ambiguous, specially considering what `timedelta` does, where `.days` for example will only return the `days` part, not convert the whole delta to `days` as a `float`.

Anyway, it is not important, I think for methods that are a simple math operation, we can make them `@property` to cut a bit the syntactic noise, but it is just a matter of taste.  Maybe it would be good to have a convention, so we don't have different styles in the SDK.

Reply:
I looked at https://google.github.io/styleguide/pyguide.html#213-properties and even when it is vague, it always talks about attributes, and I agree `as_xxx` is not really an attribute, so I agree to leave it as a function.